openvino2tensorflow/openvino2tensorflow.py

```diff
@@ -7852,7 +7865,7 @@ def main():
 
     del package_list
     os.makedirs(model_output_path, exist_ok=True)
-    convert(model, model_output_path, output_saved_model, output_h5, output_weight_and_json, output_pb,
+    convert(model, model_output_path, output_saved_model, output_h5, output_weight_and_json, output_pb, dont_save_keras_model,
```

Comment:
To be precise, `dont_convert_from_keras_model` is more correct in meaning than `dont_save_keras_model`, since we are not outputting keras_model, but only reading keras model objects.